api/dps/index_test.go

```diff
@@ -131,413 +123,387 @@ func TestIndex_Last(t *testing.T) {
+					return nil, mocks.GenericError
 				},
 			},
-			codec: &mocks.Codec{
-				UnmarshalFunc: cbor.Unmarshal,
-			},
 		}
 
-		_, err := index.Values(testHeight, testPaths)
+		_, err := index.Values(mocks.GenericHeight, mocks.GenericLedgerPaths(6))
 
 		assert.Error(t, err)
 	})
 }
 
 func TestIndex_Height(t *testing.T) {
-	testHeight := uint64(42)
-	testBlockID, err := flow.HexStringToIdentifier("98827808c61af6b29c7f16071e69a9bbfba40d0f96b572ce23994b3aa605c7c2")
-	require.NoError(t, err)
-
 	t.Run("nominal case", func(t *testing.T) {
 		t.Parallel()
 
+		codec := mocks.BaselineCodec(t)
+		codec.UnmarshalFunc = cbor.Unmarshal
+
+		id := mocks.GenericIdentifier(0)
 		index := Index{
 			client: &apiMock{
 				GetHeightForBlockFunc: func(_ context.Context, in *GetHeightForBlockRequest, _ ...grpc.CallOption) (*GetHeightForBlockResponse, error) {
-					assert.Equal(t, testBlockID[:], in.BlockID)
+					assert.Equal(t, id[:], in.BlockID)
 
 					return &GetHeightForBlockResponse{
-						BlockID: testBlockID[:],
-						Height:  testHeight,
+						BlockID: id[:],
+						Height:  mocks.GenericHeight,
 					}, nil
 				},
 			},
-			codec: &mocks.Codec{
-				UnmarshalFunc: cbor.Unmarshal,
-			},
 		}
 
-		got, err := index.HeightForBlock(testBlockID)
+		got, err := index.HeightForBlock(mocks.GenericIdentifier(0))
 
 		if assert.NoError(t, err) {
-			assert.Equal(t, testHeight, got)
+			assert.Equal(t, mocks.GenericHeight, got)
 		}
 	})
 
+	codec := mocks.BaselineCodec(t)
+	codec.UnmarshalFunc = cbor.Unmarshal
+
 	t.Run("handles index failures", func(t *testing.T) {
 		t.Parallel()
 
+		id := mocks.GenericIdentifier(0)
 		index := Index{
 			client: &apiMock{
 				GetHeightForBlockFunc: func(_ context.Context, in *GetHeightForBlockRequest, _ ...grpc.CallOption) (*GetHeightForBlockResponse, error) {
-					assert.Equal(t, testBlockID[:], in.BlockID)
+					assert.Equal(t, id[:], in.BlockID)
 
-					return nil, mocks.DummyError
+					return nil, mocks.GenericError
 				},
 			},
-			codec: &mocks.Codec{
-				UnmarshalFunc: cbor.Unmarshal,
-			},
 		}
 
-		_, err := index.HeightForBlock(testBlockID)
+		_, err := index.HeightForBlock(mocks.GenericIdentifier(0))
 
 		assert.Error(t, err)
 	})
 }
 
 func TestIndex_Transaction(t *testing.T) {
-	testTransactionID := flow.Identifier{0x98, 0x82, 0x78, 0x08, 0xc6, 0x1a, 0xf6, 0xb2, 0x9c, 0x7f, 0x16, 0x07, 0x1e, 0x69, 0xa9, 0xbb, 0xfb, 0xa4, 0x0d, 0x0f, 0x96, 0xb5, 0x72, 0xce, 0x23, 0x99, 0x4b, 0x3a, 0xa6, 0x05, 0xc7, 0xc2}
-	testTransaction := &flow.TransactionBody{
-		ReferenceBlockID: flow.Identifier{0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a},
-		Payer:            flow.Address{0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a, 0x2a},
-	}
-	testTransactionB, err := cbor.Marshal(testTransaction)
+	testTransactionB, err := cbor.Marshal(mocks.GenericTransaction(0))
```

Comment:
Again, I find the `B` confusing, as I would associate it with `A` / `B` testing. If there is nothing else, it can just be called `testBytes` imo. Or `testData`.